src/Drupal/DrupalKernel.php

```diff
@@ -203,4 +214,77 @@ protected function addDrushServiceProvider($serviceProviderName, $serviceYmlPath
             $this->serviceYamls['app'][$serviceProviderName] = $serviceYmlPath;
         }
     }
+
+    /**
+     * populates theme data on the filesystem.
+     *
+     * @see Drupal\Core\DrupalKernel::moduleData().
+     */
+    protected function themeData($theme_list)
+    {
```

Comment:
I'm not too clear why we need this code. Why is iterating over all enabled themes sufficient? Are we wanting to catch the case where Drush integration is in the parent thee? What does simpletest have to do with anything?

Reply:
I've simplified this slightly. This is necessary to populate the Drupal path cache so that drupal_get_path() and similar function work for themes. I think we can remove the simpletest stuff as we're not loading themes for testing - it is in the core `Drupal\Core\DrupalKernel::moduleData()` but reviewing it, we don't need it for this use case, so removed.